maven/impl-projects/src/main/java/org/jboss/forge/addon/maven/projects/archetype/CustomMavenArchetypeProjectType.java

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
whoops - sorry! lemme fix those - and setup an IDEA template to always use that template in forge stuff...